libs/damap/src/lib/components/dmp/doc-data-quality/doc-data-quality.component.html

```diff
@@ -74,10 +66,11 @@
     }}</mat-label>
     <input
       matInput
-      #input
+      #inputMetadata
+      [formControl]="metadata"
       maxlength="4000"
       placeholder=" As there are no domain specific metadata standards applicable, we will provide a README file with an explanation of all values and terms used [at file level/at dataset level/at project level]" />
-    <mat-hint align="end">{{ input.value.length }}/4000</mat-hint>
+    <mat-hint align="end">{{ inputMetadata.value.length }}/4000</mat-hint>
```

Comment:
Would be nice to have a reusable component or integrate it with the base [`app-textarea-wrapper`](https://github.com/tuwien-csd/damap-frontend/tree/5d7d72be6ffc0b7e93db6977bbd1e9d87972e8f2/libs/damap/src/lib/shared/textarea-wrapper) or [`app-input-wrapper`](https://github.com/tuwien-csd/damap-frontend/blob/5d7d72be6ffc0b7e93db6977bbd1e9d87972e8f2/libs/damap/src/lib/shared/input-wrapper/) as is done here:
https://github.com/tuwien-csd/damap-frontend/blob/5d7d72be6ffc0b7e93db6977bbd1e9d87972e8f2/libs/damap/src/lib/shared/input-wrapper/input-wrapper.component.html#L2-L5

Reply:
Wrong line. Should have been the tooltip :)